yt/data_objects/construction_data_containers.py

```diff
@@ -1290,35 +1290,48 @@ class YTSmoothedCoveringGrid(YTCoveringGrid):
     _min_level = None
 
     @wraps(YTCoveringGrid.__init__)
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, kind="linear", nbuf=None, **kwargs):
         ds = kwargs["ds"]
         self._base_dx = (
             ds.domain_right_edge - ds.domain_left_edge
         ) / ds.domain_dimensions.astype("float64")
         self.global_endindex = None
         YTCoveringGrid.__init__(self, *args, **kwargs)
         self._final_start_index = self.global_startindex
+        # Assign order (covergence) based on interpolation kind
+        order_nbuf = {
+            "natural": (-2, 3),
+            "akima": (-1, 3),
+            "linear": (2, 1),
+            "cubic": (4, 2),
+            "quintic": (6, 3),
+        }
+        try:
+            order, nbuf_from_order = order_nbuf[kind]
+        except KeyError as k:
+            msg = f"Interpolation '{kind}' has not been implemented. "
+            msg += "Only the following are supported: "
+            msg += ", ".join(order_nbuf.keys())
+            raise NotImplementedError(msg) from k
```

Comment:
Sorry to bother you _again_ with this. Please avoid one-letter variable names, they are much harder to grep for =)
```suggestion
        except KeyError as err:
            msg = f"Interpolation '{kind}' has not been implemented. "
            msg += "Only the following are supported: "
            msg += ", ".join(order_nbuf.keys())
            raise NotImplementedError(msg) from err
```

Reply:
Definitely no problem! Changing now.